calculateMetrics/metrics.py

```diff
@@ -0,0 +1,381 @@
+    removed_scores = []
+    username_accesser = 0
+    list_length = len(github_data)
+    for ef in range(len(github_data)):
+        print("Checking GitHub user: ", github_data[username_accesser][0])
+        # This if/else calculates the amount of code removed to the,
+        # standard deviation in order to grade the team.
+        #Ths if statment will end if the user was within 0.5 of the standard deviation.
+        if github_data[username_accesser][3] <= np.average(removed_list) + (
+            removed_sd * 0.5
+        ) and github_data[username_accesser][3] >= np.average(removed_list) - (
+            removed_sd * 0.5
+        ):
+            print("appending 5 to removed_scores list")
+            removed_scores.append(5)
+            username_accesser += 1
+        # This if statment will end if the user was within 1.0 of the standard deviation.
+        elif github_data[username_accesser][3] <= np.average(removed_list) + (
+            removed_sd * 1.0
+        ) and github_data[username_accesser][3] >= np.average(removed_list) - (
+            removed_sd * 1.0
+        ):
+            print("appending 4 to removed_scores list")
+            removed_scores.append(4)
+            username_accesser += 1
+        # This if statment will end if the user was within 1.5 of the standard deviation.
+        elif github_data[username_accesser][3] <= np.average(removed_list) + (
+            removed_sd * 1.5
+        ) and github_data[username_accesser][3] >= np.average(removed_list) - (
+            removed_sd * 1.5
+        ):
+            print("appending 3 to removed_scores list")
+            removed_scores.append(3)
+            username_accesser += 1
+        # This if statment will end if the user was within 2.0 of the standard deviation.
+        elif github_data[username_accesser][3] <= np.average(removed_list) + (
+            removed_sd * 2.0
+        ) and github_data[username_accesser][3] >= np.average(removed_list) - (
+            removed_sd * 2.0
+        ):
+            print("appending 2 to removed_scores list")
+            removed_scores.append(2)
+            username_accesser += 1
+        # This if statment will end if the user was within 2.25 of the standard deviation.
+        elif github_data[username_accesser][3] <= np.average(removed_list) + (
+            removed_sd * 2.25
+        ) and github_data[username_accesser][3] >= np.average(removed_list) - (
+            removed_sd * 2.25
+        ):
+            print("appending 1 to removed_scores list")
+            removed_scores.append(1)
+            username_accesser += 1
+        else:
+            print("Appending 0 to removed_scores list")
+            removed_scores.append(0)
+            username_accesser += 1
+    global removed_overall_score
+    removed_overall_score = np.average(removed_scores)
+    # These print statments print out the scores for the amount of code the team,
+    # removed.
+    print("")
```

Comment:
I would suggest using ` \n` instead of a blank print statement to start a new line